docs/quickstart.md

```diff
@@ -0,0 +1,139 @@
+# Quickstart
+
+This section guides you through all the necessary steps to create a workload Kubernetes cluster on top of the OpenStack infrastructure. The guide describes a path that utilizes the [clusterctl] CLI tool to manage the lifecycle of a CAPI management cluster and employs [kind] to create a local non-production management cluster.
```

Comment:
I suggest adding a link for `CAPI`

Reply:
fixed: c380e5fa26ad5d4725148dfaf08bed0f842afcc8